manifests/params.pp

```diff
@@ -44,14 +45,35 @@
 
       case $facts['os']['name'] {
         'Debian': {
+          if versioncmp($facts['os']['release']['full'], '9') == 0 {
+            $package_install_options = undef
+            $major_version_corosync_detect_by_distr = '2'
+          }
+
+          if versioncmp($facts['os']['release']['full'], '10') >= 0 {
+            $package_install_options = undef
+            $major_version_corosync_detect_by_distr = '3'
+          }
+
           if versioncmp($facts['os']['release']['full'], '8') == 0 {
             $package_install_options = ['-t', 'jessie-backports']
+            $major_version_corosync_detect_by_distr = '1'
           } else {
             $package_install_options = undef
+            $major_version_corosync_detect_by_distr = '2'
+          }
+        }
+        'Ubuntu': {
+          $package_install_options = undef
+          if versioncmp($facts['os']['release']['full'], '19.10') >= 0 {
```

Comment:
can you please add the Ubuntu LTS versions where this should work into metadata.json as well?